src/app/client/src/app/plugins/profile/components/profile-page/profile-page.component.html

```diff
@@ -306,14 +306,14 @@
                   <span class="bar mr-4"></span>
                   <span class="bar"></span>
               </div>
-              <div class="list-items__item" *ngIf="data.certificates && data.certificates.length > 0">
+              <div class="list-items__item" *ngIf="data?.certificates?.length || data?.issuedCertificates?.length">
```

Comment:
If the certificates array is empty, then this condition will not work. Check for length > 0 as well.